cmd/otelopscol/components.go

```diff
@@ -94,6 +95,7 @@ func components() (component.Factories, error) {
 		zookeeperreceiver.NewFactory(),
 		rabbitmqreceiver.NewFactory(),
 		varnishreceiver.NewFactory(),
+		saphanareceiver.NewFactory(),
```

Comment:
This looks like a new feature being enabled, not an update. Does it make sense to call this out in the PR description?

Reply:
This was added after I created the PR and I forgot to update the description after doing that. I've amended the description now.